x/projects/types/expected_keepers.go

```diff
@@ -16,3 +17,7 @@ type BankKeeper interface {
 	SpendableCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
 	// Methods imported from bank should be defined here
 }
+
+type SpecKeeper interface {
```

Comment:
why does project need specs?

Reply:
in the ValidateChainPolicies function (in the projects keeper) I check that the chain policies are configured upon actual specs that are supported by the chain

Reply:
let the user configure whatever he wants, i dont want the projects to know the specs just for that